Automattic/jetpack
# Eventbrite block: add test coverage and add input trim

Fixes https://github.com/Automattic/jetpack/issues/19088

#### Changes proposed in this Pull Request:
This PR:

- adds basic test coverage / fixtures for block and deprecations
- fixes a small input sanitization bug reported by https://github.com/Automattic/jetpack/issues/19088 and adds basic coverage for this test by abstracting trim method.

#### Jetpack product discussion
p1HpG7-b3G-p2


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

See https://github.com/Automattic/jetpack/issues/19088  for testing the input bug

Run the unit tests:
```
cd projects/plugins/jetpack
yarn fixtures:test extensions/blocks/eventbrite/test
```
